std-bits/snowflake/src/main/resources/META-INF/native-image/org/enso/snowflake/resource-config.json

```diff
@@ -6,6 +6,66 @@
             },
             {
                 "pattern": "\\Qnet/snowflake/client/jdbc/internal/apache/http/client/version.properties\\E"
+            },
+            {
+                "pattern": "/awssdk_config_override.json"
+            },
+            {
+                "pattern": "/com/amazonaws/endpointdiscovery/endpoint-discovery.json"
+            },
+            {
+                "pattern": "com/amazonaws/global/handlers/request.handler2s"
+            },
+            {
+                "pattern": "com/amazonaws/internal/config/awssdk_config_default.json"
+            },
+            {
+                "pattern": "com/amazonaws/internal/config/awssdk_config_override.json"
+            },
+            {
+                "pattern": "com/amazonaws/partitions/endpoints.json"
+            },
+            {
+                "pattern": "com/amazonaws/partitions/override/endpoints.json"
+            },
+            {
+                "pattern": "com/amazonaws/regions/override/regions.xml"
+            },
+            {
+                "pattern": "com/amazonaws/sdk/versionInfo.properties"
+            },
+            {
+                "pattern": "com/amazonaws/services/s3/request.handler2s"
+            },
+            {
+                "pattern": "com/amazonaws/services/s3/request.handlers"
+            },
+            {
+                "pattern": "com/amazonaws/util/awssdk_config_override.json"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/internal/snowflake/common/version.properties"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/internal/snowflake/common/version_en.properties"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/internal/snowflake/common/version_en_US.properties"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/version.properties"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/version_en.properties"
+            },
+            {
+                "pattern": "net/snowflake/client/jdbc/version_en_US.properties"
+            },
+            {
+                "pattern": "org/joda/time/tz/data/.*/.*"
```

Comment:
- similar global patterns
- may be responsible for #12451
- how exactly this line got here?
- is it autogenerated by the [Native Image Agent](https://www.graalvm.org/jdk24/reference-manual/native-image/metadata/AutomaticMetadataCollection/)?

Reply:
Sorry, didn't see that comment before I disappeared for a week. 

This is not autogenerated. And either agent didn't include it at all or only included it for a single tz, which had a potential to crash when dealing with a tz that I wasn't testing it on. So instead of listing all data files I used wildcards, seems more efficient and correct to me.I don't thing this is related  to that ticket.